### ISSUE TITLE
Introduce generators

### ISSUE THREAD

**Issue by minj** [minj/foxtrick#1342]
FF29+; Chrome 39+

No Safari/IE support.

**Comment by minj** [comment]
[Ubuntu 12.04LTS](http://packages.ubuntu.com/search?keywords=chromium-browser&searchon=names&suite=all&section=all) is still stuck with Chromium 37.

No point releases for 15 months now. Even though there was a 12.04.5 after 14.04 it seems [the original idea](http://www.markshuttleworth.com/archives/146) was to release them every 6 months but stop once the next LTS is available. I think it's safe to assume there will not be any more point releases for 12.04.

However, waiting for April 2017 seems ludicrous to me. People already had 20 months to upgrade to 14.04. Granted, it doesn't run well on old systems (<2004 or so) but there always are other flavors available, e.g. lubuntu 14.04 LTS.

http://gs.statcounter.com says 5.1% chrome users still have chrome < 39. This is practically identical to [the number of hopeless Firefox users](http://i.imgur.com/Ij4YYXJ.png). Incidentally, [w3schools](http://www.w3schools.com/browsers/browsers_chrome.asp) groups anything < 39 as 'Older' which amounts to 1.7%/66.5%=2.6%. Of course w3schools does not represent the general public.

In conclusion, I think it's practical to require chrome 39+ from now on. Perhaps it's time to use `minimum_chrome_version` in the manifest.

**Comment by minj** [comment]
blocked on #859

**Comment by minj** [comment]
this of course assumes there will no longer be any Safari releases 0.17.0+

**Comment by minj** [comment]
https://facebook.github.io/regenerator/

**Comment by minj** [comment]
Safari 10 supports them

**Comment by minj** [comment]
Let's not use the stop-gap.

https://caniuse.com/#search=await 72%
